tests/dummy/app/templates/demo/progress.hbs

```diff
@@ -1,52 +1,105 @@
 <div class="bs-example">
   <!-- BEGIN-SNIPPET progress-demo -->
-  {{#bs-progress as |p|}}
-    {{p.bar value=50}}
-  {{/bs-progress}}
+  <BsProgress as |p|>
+    <p.bar @value="50" />
```

Comment:
This seems to be failing in Ember 2.18, see the Travis log! Maybe it's related to `<p>` looking like a regular tag? Can you try changing this to something else, like `PG` to see if it fixes the problem?